docs/api.md

```diff
@@ -1090,6 +1092,7 @@ Fires when your hp or food change.
 
 Fires when your oxygen level change.
 
+#### "entityAttributes" (entity)
```

Comment:
Can you add a description ?

Reply:
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa